napari/_vispy/visuals/volume.py

```diff
@@ -501,6 +502,11 @@ def clim(self):
     def cmap(self):
         return self._cmap
 
+    @clim.setter
```

Comment:
ah i didn't realize this wasn't on our backport of the vispy volume, as you might see in #406 we're in process of updating to `0.6` which seems to have some clim support through it's `set_data` - https://github.com/vispy/vispy/blob/cb5706f3ec0a7298a91692f2c73edd3ea3b9dcdc/vispy/visuals/volume.py#L451

Reply:
cool! I might have to refactor then after its merged